<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelemetrySensor</class>
 <widget class="QWidget" name="TelemetrySensor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2790</width>
    <height>67</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0,0,0,1,0,0,1,1,1,0,0,1,0,1,0,1,1,1,1,1,1,0,0,0,0,1,0,0,1,0,0,0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="numLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>TELE##</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoLineEdit" name="name">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoComboBox" name="type">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="idLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Id</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoHexSpinBox" name="id">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="instanceLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Instance</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoDoubleSpinBox" name="instance">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>255.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="originLabel">
     <property name="text">
      <string>Rx</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoLineEdit" name="origin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoComboBox" name="formula">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="cellsSensorLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Cells Sensor :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoComboBox" name="cellsSensor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>---</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="AutoComboBox" name="cellsIndex">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="gpsSensorLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>GPS Sensor :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoComboBox" name="gpsSensor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>---</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="altSensorLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Alt. Sensor :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoComboBox" name="altSensor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>---</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ampsSensorLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Sensor :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoComboBox" name="ampsSensor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>---</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="AutoComboBox" name="source1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>---</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="AutoComboBox" name="source2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>---</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="AutoComboBox" name="source3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>---</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="AutoComboBox" name="source4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>---</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="AutoComboBox" name="unit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="precLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Precision</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoComboBox" name="prec"/>
   </item>
   <item>
    <widget class="QLabel" name="ratioLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Ratio</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="bladesLabel">
     <property name="text">
      <string>Blades</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoDoubleSpinBox" name="ratio">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>65535.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="offsetLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Offset</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="multiplierLabel">
     <property name="text">
      <string>Multiplier</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoDoubleSpinBox" name="offset">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>-30000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>30000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoCheckBox" name="autoOffset">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Auto Offset</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoCheckBox" name="filter">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoCheckBox" name="persistent">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Persistent</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoCheckBox" name="onlyPositive">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Positive</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoCheckBox" name="logs">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Logs</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AutoDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>autodoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>AutoComboBox</class>
   <extends>QComboBox</extends>
   <header>autocombobox.h</header>
  </customwidget>
  <customwidget>
   <class>AutoCheckBox</class>
   <extends>QCheckBox</extends>
   <header>autocheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>AutoHexSpinBox</class>
   <extends>QSpinBox</extends>
   <header>autohexspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>AutoLineEdit</class>
   <extends>QLineEdit</extends>
   <header>autolineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
